selectors/index.js

```diff
@@ -4,6 +4,8 @@ module.exports={
     password_field: '#password',
     otp: '#otp',
     otp_error: '#js-flash-container > div > div',
+    sec_key_tfa_url: 'https://github.com/sessions/two-factor/security-key',
```

Comment:
I changed this property to   sec_key_tfa_url: 'https://github.com/sessions/two-factor'

Reply:
Okay ...